### DISCUSSION TITLE
Alternative demo ideas

### DISCUSSION THREAD

**Discussion by sgbaird** [sparks-baird/self-driving-lab-demo#53] [Demo extensions and design]
## Topics
* Inorganic solid-state synthesis
* Battery formulations
* Batch chemical synthesis
* semiconductor fabrication
* polymer synthesis
* biology/artificial organ compatibility
* mobile and fixed robotic arms
* microfluidic devices
* Closed-loop microscopy
* Signals/patterns

## Materials
Explore low-cost sensors for chemistry, physics, and materials science. For example, pH sensors from Atlas Scientific.

* Packing fraction as function of ingredients and processing conditions: suspensions of particles are mixed with visually measured packing fractions. Clean out the container with vigorous mixing and then extract the waste into a waste container (all via peristaltic pumps most likely).

## Signals
* 2D signal example: vibrating a drumhead or a liquid surface and trying to match particular harmonics using speakers as inputs (similar to a Rubens tube)

* 3D signal example: measuring [WiFi strength in 3D Euclidean space](https://hackaday.com/2015/02/17/mapping-wifi-signals-in-3-dimensions/) (maybe with separate sensors) where the WiFi source positions and/or sensor positions can be adjusted, and trying to match a target distribution of the WiFi signal. It starts to become reminiscent of DFT relaxation, where multiple local optima may start to appear. See also https://www.google.com/search?q=wifi+heatmapper

**Comment by sgbaird** [comment]
@Gadgetoid, came across your profile, and very nice work with pinout.xyz.

I figured it might be worth asking, any thoughts on this project? I'm particularly interested to hear your thoughts on alternative demos (hence tagging you on this thread), modifications that you think would be interesting, and suggestions on how to maximize impact.

**Comment by iterateccvoelker** [comment]
In terms of a cheap sensor setup, you could use an infrared camera to validate/monitor reactivity of substances in terms of exothermic heat -> Optimize towards most even reaction or highest peak, reaction duration etc. 

Another one would be the stirring plate voltage/current to measure rheology.

**Comment by sgbaird** [comment]
@BAMcvoelker great suggestions!

**Comment by sgbaird** [comment]
@stefanbringuier mentioned the idea of doing color tuning of food-dyed clear 2-part epoxy in https://github.com/sparks-baird/self-driving-lab-demo/discussions/71. Nice example where it needs a way to place samples at different locations. Seems like a good companion to other liquid handler demos (e.g. [AM ARES](https://link.springer.com/article/10.1557/s43577-021-00051-1)), especially given the use of open-source syringe pump designs.

**Reply by iterateccvoelker** [reply]
What a coincidence: I had the same idea for a student project, where a robot could mix a given set of test tubes with food colors to achieve a given mix color. We would implement it in the frame of our optimization app: https://github.com/BAMresearch/WEBSLAMD

As the robot we have the EVA from automata. 

It would be nice if there was a chance to corporate on this.

**Reply by stefanbringuier** [reply]
@BAMcvoelker very cool, thanks for sharing going to definitely look through your work.

**Reply by iterateccvoelker** [reply]
@stefanbringuier Great! I look forward to connecting more.

**Comment by sgbaird** [comment]
@rekumar also mentioned the idea of an electrochemistry experiment (low-voltage, low-current), and @blaiszik mentioned some of the difficulties his team experienced automating the movement and cleaning of electrodes, changing of cells, etc. Optimizing ionic conductivity for a mixture of electrolytes is related and could also be interesting. Since many of the industry compounds for this can be quite volatile (e.g. high vapor pressures, reactivity, toxicity), using the liquid-based color-matching demo [requires chemically-resistant parts](https://github.com/sparks-baird/self-driving-lab-demo/discussions/89#discussioncomment-3887566) and hence replacement of many of the chemical handling components.

cc @CompRhys (given the connection to @Chemix-ai and batteries)

**Reply by CompRhys** [reply]
Re: electrolytes for high voltage cells we can't use aqueous electrolytes which would severely limit applicability to small/cheap proof of concept works. Flagging this -> https://chemrxiv.org/engage/chemrxiv/article-details/60c745e2bdbb894826a38afb (although you probably have seen). Aqueous more easy for demo but applications to high density i.e. high voltage cathodes reduced.

**Reply by sgbaird** [reply]
Great point. Low O2 / H2O ppm glovebox and expensive liquid handling equipment are probably necessary for the exploration of high voltage cathodes as you mentioned. I wasn't familiar with the article, but figured someone must have done something like this before. Thanks for sharing! Not sure I would have come across it otherwise.
